### DISCUSSION TITLE
feat: create projects page

### DISCUSSION THREAD

**Discussion by jaxcksn** [ttugdsc/ttugdsc#10] [Ideas]
Description: create the project page as shown in the wireframe
Reason: the projects page is needed for the website

**Comment by Guysnacho** [comment]
Alright, so the first issue would probably be an exception to this but I'd prefer if we broke these bigger issues up into smaller bite sized problems that we could tackle so we could 1. So we can see incremental changes on pages as opposed to big jumps 2. And so others also have a chance to participate (if they want to)

So do you have a particular part of the project page you want to add?
